packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -381,6 +382,9 @@ const RealAccountSignup = ({
         if (isNavigationFromExternalPlatform(routing_history, routes.binarybot)) {
             window.location = routes.binarybot;
         }
+        if (!is_closing_create_real_account_modal) {
+            setIsClosingCreateRealAccountModal(true);
+        }
```

Comment:
@suisin-deriv just need a clarification, So if user is from external platform, are we ignoring the warning modal? what will happen here?

Reply:
@amina-deriv If user is creating account in app.deriv.com we will have to show them the warning message when they try to close `create real account` modal. I think this logic will not activate the warning modal let me check on it again :pray: